conf/pipeline/pipeline.yaml

```diff
@@ -41,6 +41,7 @@ edgepipeline_register:
   cluster_dialer_endpoint: "${CLUSTER_DIALER_PUBLIC_URL}/clusteragent/connect"
   cluster_access_key: "${CLUSTER_ACCESS_KEY}"
   allowed_sources: "cdp-,fdp-"
+  edge_pipeline_access_cluster_etcd_key_prefix: "${EDGE_PIPELINE_ACCESS_CLUSTER_ETCD_KEY_PREFIX:/devops/pipeline/v2/edge/pipeline/access-key}"
```

Comment:
already in a edgepipeline_register scope.
so key should be: cluster_access_key_etcd_prefix
env should be: EDGE_PIPELINE_CLUSTER_ACCESS_KEY_ETCD_PREFIX
default: devops/pipeline/v2/edge/cluster-access-key

Reply:
adjusted